feapi/testing/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM feapi
 
-RUN apk add --no-cache --update postgresql-client
+RUN apt-get update && apt-get install -y postgresql-client-11
+RUN rm -rf /var/lib/apt/lists/*
 
 COPY requirements-test.txt /requirements-test.txt
 RUN pip install -r /requirements-test.txt
```

Comment:
I usually try not to put file in `/`. It can lead to confusion down the road.
I'd set a `WORKDIR` and do the `COPY` and `RUN` relative to that.
I like `/opt/$NAME` but there are lots of options there.